cmd/cloudinitexecute/cloudinitexecute.go

```diff
@@ -89,6 +89,22 @@ func ApplyConsole(cfg *rancherConfig.CloudConfig) {
 			log.Errorf("Failed to mount %s: %v", mount[1], err)
 		}
 	}
+
+	for _, runcmd := range cfg.Runcmd {
+		var cmd *exec.Cmd
+
+		if len(runcmd) == 0 {
+			continue
+		} else if len(runcmd) == 1 {
```

Comment:
FYI, this is redundant, if the length of a slice is X, then the slice `[X:]` is valid.  Only if you do `X+1` will you get a panic